src/App.tsx

```diff
@@ -1,24 +1,132 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useEffect,
+  useState,
+} from 'react';
+import { Header } from './components/header';
+import { Main } from './components/main';
+import { Footer } from './components/footer';
+import { Notification } from './components/notification';
+import { Todo } from './types/Todo';
+import { getTodos, deleteTodo } from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 10283;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [error, setError] = useState<boolean | string>(false);
+  const [filter, setFilter] = useState<string>('all');
+  const [loading, isLoading] = useState(false);
+  const [loadingID, setLoadingID] = useState(0);
+  const [comletedTodos, setCompletedTodos] = useState<Todo[] | null>(null);
+  let visibleTodos: Todo[] | null = todos;
+
+  if (filter === 'active') {
+    visibleTodos = todos ? todos.filter(todo => !todo.completed) : null;
+  }
+
+  if (filter === 'completed') {
+    visibleTodos = todos ? todos.filter(todo => todo.completed) : null;
   }
 
+  const handleDeleteTodo = async (todoId: number) => {
+    try {
+      setLoadingID(todoId);
+      isLoading(true);
+      await deleteTodo(todoId);
+      setTodos((visibleTodos?.filter(todo => todo.id !== todoId) || null));
+    } catch {
+      setError('delete');
+    }
+
+    isLoading(false);
+  };
+
+  const handleSetTempTodo = (todo: Todo | null) => {
+    setTempTodo(todo);
+  };
+
+  const handleSetError = (errVal: string | boolean) => {
+    setError(errVal);
+  };
+
+  const HandleSelectFilter = (filterValue: string) => {
```

Comment:
Use consistent naming convention, change 'HandleSelectFilter' to 'handleSelectFilter'.